HW_27.10/HW_27.10/ISerialization.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HW_27._10
+{
+    public interface ISerialization
+    {
+        public void Serialize(List<Book> books, string path);
```

Comment:
usually we don't use 'public' for methods in interface, because they're public by default